kuma/users/jinja2/users/ban_user_and_cleanup_summary.html

```diff
@@ -41,7 +41,37 @@
       {% else %}
       <p>{{ _('The user did not have any revisions that were reported as spam.') }}</p>
       {% endif %}
-    </div>
+    </div><!-- /#revisions-reported-as-spam -->
+
+    <h5>{{ _('Deleted:')}}</h5>
+    <div id="revisions-deleted">
+      {% if actions_taken.revisions_deleted_list %}
+        <ul>
+          {% for revision in actions_taken.revisions_deleted_list %}
+            <li>
+              {{ revision.title }}
+            </li>
+          {% endfor %}
+        </ul>
+      {% else %}
+      <p>{{ _('The user did not have any revisions that were deleted.') }}</p>
+      {% endif %}
+    </div><!-- /#revisions-deleted -->
+
+    <h5>{{ _('Reverted:')}}</h5>
+    <div id="revisions-reverted">
+      {% if actions_taken.revisions_reverted_list %}
+        <ul>
+          {% for revision in actions_taken.revisions_reverted_list %}
+            <li>
+              {{ revision.title }}
+            </li>
+          {% endfor %}
```

Comment:
For similar reasons, I think this should be a link, either to the revision or the document history.